evm_loader/program/src/instruction/account_block_add.rs

```diff
@@ -1,52 +1,50 @@
-use std::collections::BTreeSet;
-
-use crate::account::{EthereumAccount, Operator, State};
-use crate::error::{Error, Result};
-use crate::state_account::BlockedAccountMeta;
+use crate::error::Result;
 use solana_program::instruction::TRANSACTION_LEVEL_STACK_HEIGHT;
 use solana_program::{account_info::AccountInfo, pubkey::Pubkey};
 
 pub fn process<'a>(
-    program_id: &'a Pubkey,
-    accounts: &'a [AccountInfo<'a>],
+    _program_id: &'a Pubkey,
+    _accounts: &'a [AccountInfo<'a>],
     _instruction: &[u8],
 ) -> Result<()> {
     let stack_height = solana_program::instruction::get_stack_height();
     assert_eq!(stack_height, TRANSACTION_LEVEL_STACK_HEIGHT);
 
     solana_program::msg!("Instruction: Block Accounts");
 
-    let mut state = State::from_account(program_id, &accounts[0])?;
-    let operator = Operator::from_account(&accounts[1])?;
+    todo!();
```

Comment:
Do you postpone this to the other issue?

Reply:
This is postponed for later. This instruction is not used by proxy at the moment.